<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButtonStartCalc">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>生成文件md5表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>怦天乐园资源校验文件生成器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openDirButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>选择目录</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarTotal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarCurrent">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>文件进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>总进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前目录：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentDir">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前目录</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentFile">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前文件：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lableNumberOfFiles">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1 / 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInputVersion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>客户端版本：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>客户端更新描述：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Description">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>511</width>
     <height>87</height>
    </rect>
   </property>
   <property name="plainText">
    <string>新增内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>运行文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInputExec">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Paradise.exe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInputExecArgs">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>pwnsky.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>运行参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>资源CDN</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_CDN">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="plainText">
    <string>http://pd.pwnsky.com:4000/</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowCurrentDirText_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>arg[1]为 更新器版本</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
